### ISSUE TITLE
Too easy to inadvertently remove games from favorites

### ISSUE THREAD

**Issue by jjk212** [RetroPie/EmulationStation#326]
In full mode it is too easy to inadvertently remove games from the favorites.  I thought kiosk mode would do the trick, but even in that mode favorites can be removed.

I'd like to make the following suggestion:
  1. Kiosk mode not allow editing of favorite status.
  2. Possibly rework full mode to allow ROM favorite editing to be toggled on and off
      a.  Or at the very least maybe favorite removal needs a confirmation command like left, right, left or the like.

Thoughts?

**Comment by pjft** [comment]
Thanks. Just to confirm, it's not that it's been ignored, it's just that this is an isolated report for something that has been commonplace for a while, so for a change to be considered it would probably need a bit more of a critical mass. 

Two suggestions: does Kids mode handle that well? Alternatively, you could use a separate collection for the favorites so that it's not associated to the default X button behavior. 

Would these work somewhat?

**Comment by oliver** [comment]
I encountered the same problem: in Favorites list I pressed Y (mostly because the key settings on my Xbox pad are weird) and found that this immediately removes the current game from the list. This was kinda annoying.

One way to improve this might be to make this action easier to undo. Maybe don't remove the game from the list immediately but only when leaving the Favorites list (and mark it as "(removed from list)" until then)? Or remove it from the list when the user switches to the next entry in the list?

Requiring some confirmation might be a solution as well.
